android/modules/ui/src/java/ti/modules/titanium/ui/widget/TiUILabel.java

```diff
@@ -59,7 +60,7 @@ public TiUILabel(final TiViewProxy proxy)
 			protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec)
 			{
 				// Only allow label to exceed the size of parent when it's size behavior with both wordwrap and ellipsize disabled
-				if (!wordWrap && !ellipsize && layoutParams.optionWidth == null && !layoutParams.autoFillsWidth) {
+				if (!wordWrap && ellipsize != null && layoutParams.optionWidth == null && !layoutParams.autoFillsWidth) {
```

Comment:
The original logic is when wordWrap and ellipsize are both disabled aka both are false. In the changed code, the ellipsize is only checked to be not null. It no longer checks for a false value since it's now a TruncateAt and not a boolean. @manumaticx Do you think there is an impact in behavior here?